bitswap.go

```diff
@@ -363,20 +361,9 @@ func (bs *Bitswap) receiveBlocksFrom(ctx context.Context, from peer.ID, blks []b
 		bs.pm.ResponseReceived(from, combined)
 	}
 
-	// Send all block keys (including duplicates) to any sessions that want them.
-	// (The duplicates are needed by sessions for accounting purposes)
-	bs.sm.ReceiveFrom(ctx, from, allKs, haves, dontHaves)
```

Comment:
Messages are now only published to the WantRequestManager, instead of sending blocks to the Notifier and messages to the SessionManager